jetty-home/pom.xml

```diff
@@ -87,6 +87,11 @@
       <artifactId>jetty-keystore</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-maven</artifactId>
+      <optional>true</optional>
```

Comment:
do we really need this for jetty-home?

Reply:
This seems to be where all the core modules are incorporated into the distro, so yes.

Reply:
but it;s used only by the maven plugins and they are not part of the distribution so it's not needed to be included in the jetty-home.

Reply:
No, the `jetty-maven` jar is used by the module that the distro runs to deploy the unassembled webapp. See for example: https://github.com/jetty/jetty.project/blob/jetty-12.0.x-refactor-common-maven-plugin-classes/jetty-ee10/jetty-ee10-maven-plugin/src/main/resources/ee10-maven.mod

Reply:
ah right those modules which are not part of distro but added dynamically when using the maven plugin.
maybe we should add the copy of this jar in a similar way as here we have a jar which is in fact not used by any provided modules but eventually if a user is using a particular goal of a maven plugin :) 
 
mod file could be cleaned 
this doesn't exist
`lib/maven-ee10/*.jar`

Reply:
The `${jetty.base}` portion (btw you have a typo in yours) isn't necessary.
That location will be looked for in all of the configuration sources that are directories that the start.jar is aware of (eg: `${jetty.base}` then `${jetty.home}`, but you can also add more directories between those two locations via a `--add-config-dir=<dir>` command line)

Reply:
that's weird I have
```
[lib]
${jetty.base}/lib/maven-ee10/*.jar
```
And 
```
olamy@pop-os:~/dev/sources/jetty/jetty.project$ ls -lrt jetty-ee10/jetty-ee10-maven-plugin/target/it/jetty-start-war-distro-mojo-it/jetty-simple-webapp/target/jetty-base/lib/maven-ee10/
total 88
-rw-rw-r-- 1 olamy olamy 86660 Apr 26 09:14 plugin.jar
```
But sill CNFE

Reply:
I've changed the code to remove the `jetty-maven.jar` from the distro, and dynamically copied it over into the `${jetty.base}/lib/${ee}-maven` directory as is done for the `jetty-$ee}-maven-plugin.jar`.

Reply:
yeah. haha I just saw my confusion regrading the file :)